config/default.js

```diff
@@ -16,19 +16,43 @@ module.exports = {
       { name: 'Name', type: 'string', default: 'John Doe' },
       { name: 'Picture', type: 'picture' },
       { name: 'Level', type: 'number', default: 1 },
-      { name: 'Race', type: 'options', values: ['Elf', 'Human', 'Dwarf', 'Dragonite', 'Pewpew'] },
-      { name: 'HP', type: 'number', default: 20 },
-      { name: 'MaxHP', type: 'number', default: 20 },
-      { name: 'Mana', type: 'number', default: 10 },
-      { name: 'MaxMana', type: 'number', default: 10 },
+      { name: 'Race', type: 'options', values: ['Aarakocra', 'Aasimar', 'Aetherborn', 'Aven', 'Bugbear', 'Centaur', 'Changeling', 'Dragonborn', 'Dwarf', 'Elf', 'Firbolg', 'Genasi', 'Gith', 'Gnome', 'Goblin', 'Goliath', 'Half-elf', 'Half-orc', 'Halfling', 'Hobgoblin', 'Human', 'Kenku', 'Khenra', 'Kobold', 'Kor', 'Lizardfolk', 'Merfolk', 'M cont.', 'Minotaur', 'Naga', 'Orc', 'Revenant', 'Shifter', 'Siren', 'Tabaxi', 'Tiefling', 'Tortle', 'Triton', 'Vampire', 'Vedalken', 'Warforged', 'Yuan-Ti Pureblood'] },
+      { name: 'Alignment', type: 'options', values: ['Chaotic Evil', 'Chaotic Good', 'Chaotic Neutral', 'Lawful Evil', 'Lawful Good', 'Lawful Neutral', 'Neutral', 'Neutral Evil', 'Neutral Good']},
+      { name: 'Background', type: 'options', values: ['Acolyte', 'Anthropologist', 'Archaeologist', 'Black Fist Double Agent', 'Caravan Specialist', 'Charlatan', 'City Watch', 'Clan Crafter', 'Cloistered Scholar', 'Cormanthor Refugee', 'Courtier', 'Criminal/Spy', 'Dissenter', 'Dragon Casualty', 'Earthspur Miner', 'Entertainer', 'Faction Agent', 'Far Traveler', 'Folk Hero', 'Gate Urchin', 'Gladiator(Entertainer)', 'Guild Artisan', 'Guild Merchant', 'Harborfolk', 'Haunted One', 'Hermit', 'Hillsfar Merchant', 'Hillsfar Smuggler', 'Inheritor', 'Initiate', 'Inquisitor', 'Investigator', 'Iron Route Bandit', 'Knight (Noble)', 'Knight of the Order', 'Mercenary Veteran', 'Mulmaster Aristocrat', 'Noble', 'Outlander', 'Phlan Insurgent', 'Phlan Refugee', 'Sage', 'Sailor/Pirate', 'Secret Identity', 'Shade Fanatic', 'Soldier', 'Stojanow Prisoner', 'Ticklebelly Nomad', 'Trade Sheriff', 'Urban Bounty Hunter', 'Urchin', 'Uthgardt Tribe Member', 'Vizier', 'Waterdhavian Noble']},
+      { name: 'Class', type: 'options', values: ['Barbarian', 'Bard', 'Cleric', 'Druid', 'Fighter', 'Monk', 'Paladin','Ranger','Rogue','Sorcerer','Warlock','Wizard']},
+      { name: 'HP', type: 'number', default: 10 },
+      { name: 'MaxHP', type: 'number', default: 10 },
+      { name: 'Spells', type: 'spells', values: [
```

Comment:
`type` should be a generic data type/structure so that it can be implemented in a way that would work in all cases (such as text, options, number, etc). Every `type` would need to have their own custom implementation for the UI in multiple places.

Reply:
Whoops, my bad. Spells should be text. Will amend.

Reply:
Leaving location type since I think that needs to have a custom implementation?
Changed spells and proficiencies back to type 'text'.

Resolved in 5fc212e9f07ed7d7478d5a5067afd11754586fcd